site-src/geps/gep-1424.md

```diff
@@ -0,0 +1,384 @@
+# GEP-1426: HTTPRoute Mesh Binding
+
+* Issue: [#1294](https://github.com/kubernetes-sigs/gateway-api/issues/1294)
+* Status: Implementable
```

Comment:
I am not saying _no_ to marking this _implementable_ right away, but I did want to take a moment to point it out and just ask a couple questions to those who are thinking about building out the implementation. Marking this `Implementable` is meant to indicate that anyone thereafter can make a PR to add the implementation to the API. Is this actually the state we're in, and do we want that? Alternatively, do we want it `Provisional` to give others a bit more time to add more iterative PRs on top of this, perhaps honing in and further expanding and refining some of the key points?

Again, not a blocker because end of the day I don't want to slow down our progress towards an alpha, and even if we had to make significant changes in the alpha that's OK. More just a double tap because once you really get the momentum of an implementation going the refinement process can become a bit slower.

Reply:
I don't consider this a blocker, but I would ask that unless there's a strong reason to not mark this as provisional, we mark it provisional before we merge.